### PR TITLE
services/horizon/internal/docs: Update /fee_stats with max_fee and fee_charged.

### LINKED ISSUE #1990: Update /fee_stats docs
Fee stats docs should be updated with the changes: https://github.com/stellar/go/pull/1964

### PR DESCRIPTION
**Pull request by abuiles**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
* [ ] I've updated any docs ([developer docs](https://www.stellar.org/developers/reference/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [ ] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [ ] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

### What

Update fee stats docs with changes from  #1964. Fixes #1990.

### Why

PR #1964 added new fields. 

### Known limitations

N/A

### CODE REVIEW

File: services/horizon/internal/docs/reference/endpoints/fee-stats.md [hunk 1]
```diff
@@ -29,18 +31,65 @@ Response contains the following fields:
...
-| p30_accepted_fee | 30th percentile accepted fee in the last 5 ledger. |
-| p40_accepted_fee | 40th percentile accepted fee in the last 5 ledger. |
-| p50_accepted_fee | 50th percentile accepted fee in the last 5 ledger. |
-| p60_accepted_fee | 60th percentile accepted fee in the last 5 ledger. |
-| p70_accepted_fee | 70th percentile accepted fee in the last 5 ledger. |
-| p80_accepted_fee | 80th percentile accepted fee in the last 5 ledger. |
-| p90_accepted_fee | 90th percentile accepted fee in the last 5 ledger. |
-| p95_accepted_fee | 95th percentile accepted fee in the last 5 ledger. |
-| p99_accepted_fee | 99th percentile accepted fee in the last 5 ledger. |
+| mode_accepted_fee | Mode max fee in the last 5 ledger. |
```
Comment (ire-and-curses) [inline services/horizon/internal/docs/reference/endpoints/fee-stats.md]: I see this was here before this change, but can we change all `ledger` to `ledgers`?

Comment (ire-and-curses) [inline services/horizon/internal/docs/reference/endpoints/fee-stats.md]:
```suggestion
| mode_accepted_fee | Mode (most common value) of the maximum fee bid over the last 5 ledgers. |
```

File: services/horizon/internal/docs/reference/endpoints/fee-stats.md [hunk 2]
```diff
@@ -29,18 +31,65 @@ Response contains the following fields:
...
+represents the actual fee paid for the transaction while `max_fee` represents
+the maximum bid the transaction creator was willing to pay for the transaction.
+
+### Max Fee Object
+
+Information about max fee bid for transactions in the last 5 ledgers.
+
+| Field | |
+| - | - |
+| min | Minimum max fee in the last 5 ledger. |
```
Comment (ire-and-curses) [inline services/horizon/internal/docs/reference/endpoints/fee-stats.md]:
```suggestion
| min | Minimum (lowest) value of the maximum fee bid over the last 5 ledgers. |
```

File: services/horizon/internal/docs/reference/endpoints/fee-stats.md [hunk 3]
```diff
@@ -29,18 +31,65 @@ Response contains the following fields:
...
-| p40_accepted_fee | 40th percentile accepted fee in the last 5 ledger. |
-| p50_accepted_fee | 50th percentile accepted fee in the last 5 ledger. |
-| p60_accepted_fee | 60th percentile accepted fee in the last 5 ledger. |
-| p70_accepted_fee | 70th percentile accepted fee in the last 5 ledger. |
-| p80_accepted_fee | 80th percentile accepted fee in the last 5 ledger. |
-| p90_accepted_fee | 90th percentile accepted fee in the last 5 ledger. |
-| p95_accepted_fee | 95th percentile accepted fee in the last 5 ledger. |
-| p99_accepted_fee | 99th percentile accepted fee in the last 5 ledger. |
+| mode_accepted_fee | Mode max fee in the last 5 ledger. |
+| p10_accepted_fee | 10th percentile max fee in the last 5 ledger. |
```
Comment (ire-and-curses) [inline services/horizon/internal/docs/reference/endpoints/fee-stats.md]: Consider changing `in` to `over` in all these lines.

Comment (abuiles) [reply services/horizon/internal/docs/reference/endpoints/fee-stats.md]: thanks!

File: services/horizon/internal/docs/reference/endpoints/fee-stats.md (hunk 1 above)
Comment (abuiles) [reply services/horizon/internal/docs/reference/endpoints/fee-stats.md]: sure

**Pull request merged**